src/include/alloc.h

```diff
@@ -88,8 +88,10 @@ alloc_env_var_cb(const struct niova_env_var *nev);
 
 struct niova_vbasic_allocator
 {
+    unsigned int nvba_alignment;
```

Comment:
Let's make `nvba_unit_size` a uint32 and have the init functions ensure that the `unit_size` fits into 32-bits.